.eslintrc.json

```diff
@@ -17,12 +17,10 @@
     "plugin:prettier/recommended"
   ],
   "rules": {
-    "curly": ["error", "multi-line", "consistent"],
```

Comment:
In the project I created this for, curly was not enabled by default. So this might make things less-strict. Could you verify this?

Reply:
It will make things less strict. The problem is with the `multi-line` option: Prettier might split a line due to length, and then what was a `curly` pass is now a `curly` fail. How do you feel about the `all` option?

Reply:
Is this really a problem?

Initial code

```
if (condition) long_expression()
```

After first prettier run

```
if (condition)
  long_expression()
```

After curly auto-fixer

```
if (condition) {
  long_expression()
}
```

So it should stabilize after 2 file saves.

-------

curly `all` is basically what we have in tslint right now. Sometimes I find it unnecessary that very short statemets are forced to take 3 lines instead of 1. But I got used to it.

The only scenario we definitely need to avoid is

```
if (condition)
  a()
```

which gets easily extended to

```
if (condition)
  a()
  b()
```

The problem is not so big with auto-formatting, but still.

Reply:
I'll set it to `all` so we don't need to worry about saving twice/order effects, and if we decide the extra lines are annoying we can change it later.

Reply:
Perfect, feel free to merge when done and green